docs/recipes/make-stream-from-buffer.md

```diff
@@ -79,13 +79,14 @@ gulp.task('write-versions', function() {
   availableVersions.forEach(function(v) {
     // make a new stream with fake file name
     var stream = source('final.' + v);
+    
+    var streamEnd = stream;
```

Comment:
@jmm I added an extra commit.
The assignment inline 83 is now being used.
The purpose here is to show that you should always pipe to the end of the stream.